packages/nimble-components/src/table-column/icon/index.ts

```diff
@@ -21,6 +27,9 @@ import { MappingIconConfig } from '../enum-base/models/mapping-icon-config';
 import { MappingSpinnerConfig } from '../enum-base/models/mapping-spinner-config';
 import { MappingText } from '../../mapping/text';
 import { MappingTextConfig } from '../enum-base/models/mapping-text-config';
+import { TableColumnMappingWidthMode } from './types';
+
+const iconSizeWidth = columnIconSize + 2 * columnSpacing;
```

Comment:
When this calculation is moved out can you give a name to the `2`. It's not immediately clear from source what that is for. Just extra padding?